pkg/results/results.go

```diff
@@ -12,25 +12,23 @@ import (
 	"github.com/open-policy-agent/opa/rego"
 )
 
-type outputResult map[string]interface{}
+type outputResult map[string][]string
 
 // Result holds the information about the result of a check
 type Result struct {
 	ID      string
-	Value   interface{}
+	Value   []string
```

Comment:
I'd rather introduce a `Success bool` property and a `Violations []string` one, and deprecate `Value`. Although I'd keep `Value` in place, but documented as deprecated.

Reply:
Would you rather a `Success bool` over just adding Violations and having the Success state depend on the presence of Violations?

Reply:
We can omit the `Success bool`, I'm fine with that. I'd keep the `IsSuccessState` and `IsFailureState` functions.

Reply:
:+1: I've kept those and based them on whether or not the violations list is empty or not.